src/Message/CIMResponse.php

```diff
@@ -7,6 +7,11 @@
  */
 class CIMResponse extends CIMAbstractResponse
 {
+    /**
+     * For Error codes: @see https://developer.authorize.net/api/reference/responseCodes.html
+     */
+    const ERROR_RESPONSE_CODE_CANNOT_ISSUE_CREDIT = 54;
```

Comment:
Ah understood. Fixed in https://github.com/xola/authorizenet/commit/c7d9a7197eb5bca40dec726ac6f3e1dd1486ff56